### PR TITLE
Added the RGB controls to the smart dashboard and tuned them to look …

### PR DESCRIPTION
**Pull request by HarlanHaller**
…like yellow

### CODE REVIEW

**Review by christianr-172 — APPROVED** [review]
Looks good

**Comment by tagnihotri** [comment]
I'm sure its good

**Comment by vwahlig** [comment]
Super fun! Thanks for sharing!

**Review by vwahlig — APPROVED** [review]
makes sense

**Review by FRCTeam172 — APPROVED** [review]
Maybe create constants for different colors

**Review by vwahlig — APPROVED** [review]

File: .wpilib/wpilib_preferences.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
     "enableCppIntellisense": false,
     "currentLanguage": "java",
     "projectYear": "2023Beta",
-    "teamNumber": 9172
```
Comment (vwahlig) [inline .wpilib/wpilib_preferences.json:5]: i have a question

**Review by ehunthawkins — APPROVED** [review]

**Review by tagnihotri — APPROVED** [review]

**Pull request merged**